### ISSUE TITLE
Object naming issue in browser bundle

### ISSUE THREAD

**Issue by kind3r** [GetStream/stream-chat-react#432]
While trying to create a concept chat component to be embedded in an existing website, I discovered that `stream-chat-react` can't be used as an external dependecy as all the objects are exported to the global namespace overwriting at least one of the browser's built in objects such as [**File**](https://developer.mozilla.org/en-US/docs/Web/API/File#:~:text=A%20File%20object%20is%20a,Blob%20s%20and%20File%20s.).

Since renaming this object (to something like `UploadedFile`) I would presume to be out of the question, a simple solution would be to export the objects under a [custom namespace other than `window`](https://github.com/GetStream/stream-chat-react/blob/26b505b201b8cde05d5240863fd71931a9e28172/rollup.config.js#L155).

**Comment by jaapbakker88** [comment]
@kind3r can you elaborate a little bit. Are you using the browser bundle? If yes, can you create an example on codepen for me to checkout?

**Comment by kind3r** [comment]
@jaapbakker88 , 

You can use any of the provided examples like https://codepen.io/team/stream/pen/pYrMJe and just change the browser bundle version to 1.2.5 (https://cdn.jsdelivr.net/npm/stream-chat-react@1.2.5/dist/browser.full-bundle.min.js)

**Comment by kind3r** [comment]
You'll see it fails the following check in `seamless-immutable` used by `stream-chat`:

```
  function isFileObject(obj) {
    return typeof File !== 'undefined' &&
           obj instanceof File; <--- HERE
  }
```

**Comment by mahboubii** [comment]
@kind3r fixed in [v2.1.0](https://github.com/GetStream/stream-chat-react/releases/tag/v2.1.0)
